hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -607,6 +613,22 @@ public void removeServers(Set<Address> servers) throws IOException {
     return result;
   }
 
+  private Map<ServerName, List<RegionInfo>> getAssignmentsByGroup(String groupName)
```

Comment:
mis-placed regions will not be moved to the group.
We should get the grouping information through the tables in the group instead of the servers

Reply:
Now balanceRSGroup will perform load balancing of group server dimensions.
Suppose we have two groups(groupA and groupB), if tableA's region mis-placed on groupB, we can execute balanceRSGroup(GroupB) to move it back to the right place. @guangxuCheng

Reply:
Yes, we can balance other groups, but if there are many mis-placed regions, we may need to balance many groups, which I think is unreasonable.
In addition, before the modification, all mis-placed regions of the group can be moved to the correct group by balancing the group.